README.md

```diff
@@ -11,14 +11,19 @@ Schrodinger intends to continue to contribute to this code as it still uses it i
 
 Examples and documentation will be added/improved over time
 
+## Templates
+
+Coordgen uses templates for some macrocycle systems. The default templates are in `templates.mae`. If you need additional
+local templates, use `CoordgenTemplates::setTemplateDir()`, and put them in a file called `user_templates.mae`.
```

Comment:
This is now outdated, right?

We should mention that, by default, the templates are now hardcoded and that templates.mae does not get installed, unless we set `COORDGEN_USE_MAEPARSER`, which keeps the old behavior (install/use templates). Maybe also mentioned that the tests also use the templates file.

Reply:
I still think we should clarify that templates are now hardcoded, and `templates.mae` is provided mostly for testing.

Reply:
I don't think this is outdated. The templates are in templates.mae. It's just that templates.mae is parsed to generate data that is in the library. If someone wanted to visualize the templates that we actually use, reading/displaying "templates.mae" would be the right choice. If someone wanted to update the templates that we use, editing "templates.mae" would be the right choice.

If this were internal, I'd build CoordgenTemplates.h and CoordgenTemplates.cpp at build time instead of at run time. I poke around at generating those files using maeparser.cpp at compile time, but that seems perhaps overly complex.

I'll add a comment about how to update templates.